src/js/disability-benefits/526EZ/config/form.js

```diff
@@ -21,55 +24,43 @@ import {
   facilityDescription,
   treatmentView,
   recordReleaseWarning,
-  validateAddress,
+  // validateAddress, // TODO: This needs to be fleshed out
   documentDescription,
   evidenceSummaryView,
   additionalDocumentDescription,
   releaseView
 } from '../helpers';
 
 const {
-  treatments
+  treatments,
+  privateRecordReleases
 } = fullSchema526EZ.properties;
 
 const {
   date,
-  // files
+  dateRange,
+  privateTreatmentCenterAddress
 } = fullSchema526EZ.definitions;
 
 const FIFTY_MB = 52428800;
 
-// We may add these back in after the typeahead, but for now...
-const treatmentsSchema = _.set('items.properties.treatment.properties',
-  _.omit(
-    [
-      'treatmentCenterType',
-      'treatmentCenterCountry',
-      'treatmentCenterState',
-      'treatmentCenterCity'
-    ],
-    treatments.items.properties.treatment.properties
-  ), treatments);
+// TODO: Remove once typeahead supports auto-filling address and treatment center type
+const vaTreatments = ((treatmentsCommonDef) => {
+  const { type, maxItems, items } = treatmentsCommonDef;
 
-const privateRecordReleasesSchema = Object.assign({}, treatments.items.properties.treatment.properties, {
-  privateMedicalRecordsReleaseAccepted: {
-    type: 'boolean'
-  },
-  'view:privateMedicalRecordsReleasePermissionRestricted': {
-    type: 'object',
-    'ui:collapsed': true,
-    properties: {}
-  },
-  treatmentCenterStreet1: {
-    type: 'string'
-  },
-  treatmentCenterStreet2: {
-    type: 'string'
-  },
-  treatmentCenterPostalCode: {
-    type: 'number'
-  }
-});
+  return {
+    type,
+    maxItems,
+    items: {
+      type: items.type,
+      properties: _.omit(
+        ['treatmentCenterAddress', 'treatmentCenterType'],
+        items.properties
+      )
+    }
+  };
+
+})(treatments);
```

Comment:
Could we use _.set here instead, if we're only planning to change one property?

Reply:
I think this is referring to an older version of the code? Lemme know if not.